client/src/helpers.js

```diff
@@ -60,6 +61,20 @@ class Helpers {
   isFunction(item) {
     return typeof item === 'function';
   }
+
+  getLuigiCoreDomain() {
+    return this.origin;
+  }
+
+  setLuigiCoreDomain(origin) {
+    if (origin) {
+      this.origin = origin;
+    }
+  }
+
+  sendPostMessageToLuigiCore(msg) {
```

Comment:
There are still two places where we are not using this method and still sending (`luigi.navigate` listener), please update those calls.